### DISCUSSION TITLE
Bridge tx DAI -> wDAI stuck

### DISCUSSION THREAD

**Discussion by danielDeFi** [wormhole-foundation/wormhole#214] [Problems / bugs] [answered]
Hello team,

I used wormholebridge.com to bridge DAI ERC20 tokens to wDAI SLP tokens. The tx got approved on the Ethereum chain, but wDAI tokens were never minted on Solana.

TX hash on Ethereum chain: 0xdcf2cad4af3caf1a7b721fe8b877f9fe89f1bd23fdf9b65903b56dc986913699
Receiving Solana address: 2kd9pBHpZmFrMqp9ZVj6nDcavwYugQLiu17seAcSqnRM

Could you please help with finalizing this bridge tx?

Thanks in advance.

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed in https://explorer.solana.com/tx/2oxSdRELCMxrUPTnAPZoTmtyh9aJe5jWXGWeLq2fAjzW22ACjKqzbDdymyEEcz8nGyFsyGY2gMTyN4FRFUPdtguJ

**Reply by danielDeFi** [reply]
Thanks Hendrik!
